package.json

```diff
@@ -24,36 +24,39 @@
     "bitcoin",
     "BTC"
   ],
+  "author": "Bitfinex",
   "contributors": [
     "Ezequiel Wernicke <ezequiel.wernicke@bitfinex.com> (https://www.bitfinex.com)",
     "Josh Rossi <josh@bitfinex.com> (https://www.bitfinex.com)",
     "Cris Mihalache <cris.m@bitfinex.com> (https://www.bitfinex.com)",
     "Robert Kowalski <robert@bitfinex.com> (https://www.bitfinex.com)",
     "Simone Poggi <simone@bitfinex.com> (https://www.bitfinex.com)",
-    "Paolo Ardoino <paolo@bitfinex.com> (https://www.bitfinex.com)"
+    "Paolo Ardoino <paolo@bitfinex.com> (https://www.bitfinex.com)",
+    "Jacob Plaster <jacob.plaster@bitfinex.com> (https://www.bitfinex.com)"
   ],
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/bitfinexcom/bfx-api-node-rest/issues"
   },
   "devDependencies": {
     "babel-cli": "^6.26.0",
+    "babel-eslint": "^10.0.3",
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-plugin-transform-regenerator": "^6.26.0",
     "babel-preset-env": "^1.7.0",
-    "bfx-api-mock-srv": "git+https://github.com/bitfinexcom/bfx-api-mock-srv.git",
-    "chai": "^3.4.1",
+    "bfx-api-mock-srv": "^1.0.0",
```

Comment:
- [ ] Publish `bfx-api-mock-srv` to npm before merge

Or it might be better to reference `git+https://github.com/bitfinexcom/bfx-api-mock-srv.git#v1.0.0`, that way we can get write access off paolo and update without him needing to publish to npm. Ill let you decide

Reply:
I'll await publish of `bfx-api-mock-srv`